test/unit/account_test.js

```diff
@@ -1,27 +1,43 @@
-
 describe('Account.constructor', function() {
-  it("fails to create Account object from an invalid address", function() {
-    expect(() => new StellarBase.Account('GBBB')).to.throw(/accountId is invalid/);
+  const ADDRESS = 'GBBM6BKZPEHWYO3E3YKREDPQXMS4VK35YLNU7NFBRI26RAN7GI5POFBB';
+  const MUXED_ADDRESS =
+    'MA7QYNF7SOWQ3GLR2BGMZEHXAVIRZA4KVWLTJJFC7MGXUA74P7UJVAAAAAAAAAAAAAJLK';
+
+  it('fails to create Account object from an invalid address', function() {
+    expect(() => new StellarBase.Account('GBBB')).to.throw(
+      /accountId is invalid/
+    );
+  });
+
+  it('fails to create Account object from an invalid sequence number', function() {
+    expect(() => new StellarBase.Account(ADDRESS, 100)).to.throw(
+      /sequence must be of type string/
+    );
```

Comment:
What about an invalid string?

Reply:
This should be covered by BigNumber, but I've added a test case :heavy_check_mark: